site/content/en/docs/Guides/Client SDKs/unreal.md

```diff
@@ -75,4 +75,139 @@ Available settings:
 - Health Ping Seconds. Interval of the server sending a health ping to the Agones sidecar. (default: `5`)
 - Request Retry Limit. Maximum number of times a failed request to the Agones sidecar is retried. Health requests are not retried. (default: `30`)
 - Send Ready at Startup. Automatically send a Ready request when the server starts. Disable this to manually control when the game server should be marked as ready. (default: `true`)
+{{% /feature %}}
+{{% feature publishVersion="1.8.0" %}}
+# Agones Unreal SDK
```

Comment:
I think we can drop this top level title, as you can see:
https://aac2629-dot-preview-dot-agones-images.appspot.com/site/docs/guides/client-sdks/unreal/

The layout starts to look a little odd.

Reply:
Oh this is cool, didnt realise I could see the generated docs :)